src/include/daos_srv/daos_engine.h

```diff
@@ -547,12 +550,167 @@ static inline int
 dss_abterr2der(int abt_errno)
 {
 	switch (abt_errno) {
-	case ABT_SUCCESS:	return 0;
-	case ABT_ERR_MEM:	return -DER_NOMEM;
-	default:		return -DER_INVAL;
+	case ABT_SUCCESS:
+		return 0;
+	case ABT_ERR_MEM:
+		return -DER_NOMEM;
+	default:
+		return -DER_INVAL;
+	}
+}
+
+/* Convert DAOS errno to Argobots ones. */
+static inline int
+dss_der2abterr(int der)
+{
+	switch (der) {
+	case -DER_SUCCESS:
+		return ABT_SUCCESS;
+	case -DER_NOMEM:
+		return ABT_ERR_MEM;
+	default:
+		return ABT_ERR_OTHER;
 	}
 }
 
+/** Helper converting ABT error code into human readable string */
+static inline const char *
+dss_abterr2str(int rc)
+{
+	static const char *err_str[] = {"ABT_SUCCESS",
```

Comment:
[Question] It's interesting to see a `static` in an `inline`... :) Perhaps it wouldn't hurt to avoid this by not inlining or placing `err_str` outside? What do you think?

Reply:
Good catch.
I will change  this.

- [ ] Not mix inline func with static var

Reply:
Fixed with commit 0b0b9497c